src/setuptools_scm/git.py

```diff
@@ -111,16 +111,29 @@ def parse(
         dirty = wd.is_dirty()
 
         if rev_node is None:
-            return meta("0.0", distance=0, dirty=dirty, config=config)
-
-        return meta(
-            "0.0",
-            distance=wd.count_all_nodes(),
-            node="g" + rev_node,
-            dirty=dirty,
-            branch=wd.get_branch(),
-            config=config,
-        )
+            if config.fallback_version:
```

Comment:
you could make this a little bit cleaner with
```
fallback_version = "0.0"

if config.fallback_version:
  fallback_version = config.fallback_version
```
... then use `fallback_version` instead of `"0.0"`.

Reply:
Also consider adding `expandvars(fallback_version)`

Reply:
> expandvars

why need expandvars?

Reply:
to use environment variables in the fallback_version, but that could be a different change